### PR TITLE
home basic

### PR DESCRIPTION
**Pull request by teja538**
added basic rounting in the navbar
added home page
=> looking forward to suggestions to improve

deployed netlify link : https://deploy-preview-1--courageous-parfait-f47d9c.netlify.app/ 

Screenshot (282)
Screenshot (283)

### CODE REVIEW

**Review by sashaboi — APPROVED** [review]
Everything looks amazing. You might wanna split your auth pages into folders as well when you add css later. 

looking forward to seeing progress.

**Comment by rishichourasia** [comment]
Your Live preview is not working , although Screenshots works just fine . Project structure seems ok to me since there's not much to review . keep up the goood work

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
 import './App.css';
 
+import {Home} from "./pages/home/home.jsx";
+import {Cart} from "./pages/cart/cart.jsx";
+import {WishList} from "./pages/wish-list/wish-list.jsx";
+import {ProductListing} from "./pages/product-listing/product-listing.jsx";
+import {Nav} from "./components/nav.jsx";
+import {Login} from "./pages/auth/login.jsx";
+import {SignUp} from "./pages/auth/sign-up.jsx"
+
```
Comment (dusariudaykumar) [inline src/App.js:11]: You can use an ```index.js``` file and add all your exports here and you can simply import all your component from a single file.

**Review by dusariudaykumar — APPROVED** [review]
The code is clean and great folder structure.

**Pull request merged**